apis/nucleus/src/components/listbox/components/ListBoxRowColumn/components/ListBoxRoot.jsx

```diff
@@ -144,10 +145,10 @@ const RowColRoot = styled('div', {
     justifyContent: 'flex-end',
     ...ellipsis,
     width: 'auto',
-    maxWidth: 'fit-content',
-    minWidth: 'fit-content',
+    maxWidth: 'max-content',
+    minWidth: 'max-content',
     textAlign: 'right',
-    padding: '2px',
+    paddingLeft: '2%',
```

Comment:
will not 2% become to large for a very wide listbox?

Reply:
It won't be noticed so much in that case (since it's so wide). But let's review it again 👍

Reply:
I will use smaller padding in grid mode for now (can be improved upon).